FordLabs/PeopleMover
# Add link to people mover logo

## Issue
Connects #208

## What was done
- [x] Added a link around the `PeopleMover` logo that directs to the dashboard.

## How to test
1. Go to a space and click on the people mover logo.
2. Ensure it directs you to the dashboard.
